jobs: TestPauseUnpauseJob failed

jobs.TestPauseUnpauseJob [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7558553?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7558553?buildTab=artifacts#/) on master @ [ea7c52e5a930be203318db6e338edd5dde7537bd](https://github.com/cockroachdb/cockroach/commits/ea7c52e5a930be203318db6e338edd5dde7537bd):

Fatal error:

```
panic: test timed out after 4m55s

```
Stack: 

```
goroutine 125242 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0xbb
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x48
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestPauseUnpauseJob
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/000f42e66f4bc5220cdea84be2328068/logTestPauseUnpauseJob1148325739
    test_log_scope.go:79: use -show-logs to present logs inline
```
</p>
</details>
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/jobs
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestPauseUnpauseJob.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-21540

--- a/pkg/jobs/BUILD.bazel
+++ b/pkg/jobs/BUILD.bazel
@@ -78,7 +78,7 @@ go_library(
 
 go_test(
     name = "jobs_test",
-    size = "medium",
+    size = "large",
     srcs = [
         "delegate_control_test.go",
         "executor_impl_test.go",
@@ -93,7 +93,7 @@ go_test(
         "scheduled_job_test.go",
         "testutils_test.go",
     ],
-    args = ["-test.timeout=295s"],
+    args = ["-test.timeout=895s"],
     embed = [":jobs"],
     shard_count = 16,
     deps = [

